packages/cx-navigation/src/components/MenuToggler/MenuTogglerClean.tsx

```diff
@@ -12,31 +12,16 @@
  * limitations under the License.
```

Comment:
tbh-i question whether we need this component at all - it's just an icon inside a wrapper - Could probably just be a slot in the burger menu and use withChild to add the appropriate icon...